same-origin-policy/same-origin-policy.markdown

```diff
@@ -745,7 +745,7 @@ assert Confidentiality {
 }
 ```
 
-The integrity property is the dual of confidentiality: 
+The integrity property is the double of confidentiality: 
```

Comment:
I wasn't entirely sure what was meant here, but if you mean "the twin of" or "the integrity property is just like confidentiality", then "double" works better than "dual".